src/FSharpVSPowerTools.Core/UntypedAstUtils.fs

```diff
@@ -1149,11 +1150,58 @@ module Outlining =
               yield! collectOpens decls
               yield! Seq.collect parseDeclaration decls }
 
-    let getOutliningRanges tree =
+    type private LineNum = int
+    type private LineStr = string
+    type private MultilineComment = (LineNum * LineStr) list
```

Comment:
Is this really `MultilineComment` (comments between (* and *) )? I think it's more like `CommentList`.